src/components/progress/progress.tsx

```diff
@@ -44,11 +45,14 @@ export type EuiProgressProps = CommonProps & {
   size?: EuiProgressSize;
   color?: EuiProgressColor;
   position?: EuiProgressPosition;
+  /** Describe the type of element to render */
+  Element?: 'progress' | 'meter';
```

Comment:
Props should always be lowercase camel case. 
```suggestion
  /** Which HTML element to render */
  element?: 'progress' | 'meter';
```